data.json

```diff
@@ -647,6 +647,15 @@
     "dateAdded": "2019-01-25T06:45:29.000Z",
     "tags": ["expired", "stickers"]
   },
+  {
+    "name": "NativeScript",
+    "difficulty": "medium",
+    "description": "Make your mark on Native Script! Either build a neat project on StackBlitz, or contribute a quality PR. Tweet about your work and mention @NativeScript. Be sure to DM us your address to claim your NativeScript Stickers!!",
+    "reference": "https://twitter.com/NativeScript/status/1709066192605061392",
+    "image": "https://i.imgur.com/jrwefsD.jpg",
```

Comment:
Can you please choose another image so it renders ok in the preview ? 🙏

Reply:
I will do it now